utils/conversions.ts

```diff
@@ -246,8 +217,8 @@ export const toEligibilityCheckFromDSU = (
         has_discrepancies:
           data.DatiIndicatore.PresenzaDifformita === SiNoTypeEnum.SI,
         isee_type: data.DatiIndicatore.TipoIndicatore,
-        max_amount: bonusValue,
-        max_tax_benefit: calculateMaxBonusTaxBenefit(bonusValue),
+        max_amount: bonusValue.max_amount,
```

Comment:
@gunzip this doesn't work as `bonusValue.max_amount` can be `undefined`, but if I make `max_amount` required in the API specs, then the default mechanism will not work anymore

Reply:
yes the best thing (which breaks backward compatibility) is to add a new required field 

```
...
amount:
   $ref: "#/definitions/MaxBonusAmount"  
```